docs/src/installation_upgrade.md

```diff
@@ -236,6 +236,179 @@ When versions are not directly upgradable, the old version needs to be
+value, almost all the cases we have examined during support incidents or
+community issues show that this value is left unchanged.
+
+The [new default value is 1800 seconds](https://github.com/cloudnative-pg/cloudnative-pg/commit/9f7f18c5b9d9103423a53d180c0e2f2189e71c3c),
+the equivalent of 30 minutes.
+
+The new `smartShutdownTimeout` parameter has been introduced to define
+the maximum time window within the `stopDelay` value, reserved to gracefully
+stop PostgreSQL using the `smart` shutdown procedure.  Once elapsed, the
+remaining time up to `stopDelay` will be reserved for PostgreSQL to complete
+its duties regarding WAL commitments with both the archive and the streaming
+replicas to ensure the cluster doesn't lose any data.
+
+If you want to retain the old behavior, you need to set explicitly:
+
+```yaml
+spec:
+   ...
+   stopDelay: 30
+```
+
+And, **after** the upgrade has completed, specify `smartShutdownTimeout`:
+
+```yaml
+spec:
+   ...
+   stopDelay: 30
+   smartShutdownTimeout: 15
+```
+
+!!! Important
+    This change has been backported to all supported minor releases. As a
+    result, it will also be available in versions 1.20.3 and 1.19.5.
+
+#### Delay for PostgreSQL startup
+
+Until now, [the `startDelay` parameter](instance_manager.md#startup-liveness-and-readiness-probes)
+was set to 30 seconds, and CloudNativePG was using this parameter as the initial
+delay for the Kubernetes liveness probe. Given that all the supported Kubernetes
+releases provide [startup probes](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-startup-probes),
+version 1.21 has adopted this approach as well (`startDelay` is now
+automatically divided into periods of 10 seconds of duration  each).
+
+!!! Important
+    In order to add the `startupProbe`, each pod needs to be restarted.
+    As a result, when you upgrade the operator, a one-time rolling
+    update of the cluster will be executed even in the online update case.
+
+Despite the recommendations to change and tune this value, almost all the cases
+we have examined during support incidents or community issues show that this
+value is left unchanged. Given that this parameter influences the startup of
+a PostgreSQL instance, a low value of `startDelay` would cause Postgres
+never to reach a consistent recovery state and be restarted indefinitely.
+
+For this reason, `startDelay` has been [raised by default to 3600 seconds](https://github.com/cloudnative-pg/cloudnative-pg/commit/4f4cd96bc6f8e284a200705c11a2b41652d58146),
+the equivalent of 1 hour.
+
+You can achieve similar behavior using the startup probe instead of
+liveness probes with an initial delay by explicitly setting:
+
+```yaml
+spec:
+   ...
+   startDelay: 30
+```
+
+!!! Important
+    This change has been backported to all supported minor releases. As a
+    result, it will also be available in versions 1.20.3 and 1.19.5.
+
```

Comment:
hierarchically, this is under the "Upgrading to 1.21" section.
This warning means that this is not accurate.
I think we probably want to make that one "Upgrading to 1.21, 1.20.3 or 1.19.5",
And then have a sub-section, or a box, for 1.21 only.

Reply:
And. What happens if I upgrade from a previous release to 1.19.5? The warnings in this section apply, definitely. And upgrading from 1.19.5 to 1.21 should then be smooth sailing.

Reply:
This makes sense. Please check my new version. Thanks for pointing this out.